### PR TITLE
[fr] translate task "Declare Network Policy"

### PR DESCRIPTION
**Pull request by k0rventen**
Hi,

This adds the French translation for the  "Declare Network Policy" task from the "Administer a Cluster" section: https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy/.

I also added the supporting file and translated i18n strings regarding third party content that were used in this task.

### CODE REVIEW

File: data/i18n/fr/fr.toml [hunk 1]
```diff
@@ -151,5 +151,19 @@ other = "Fin du support:"
...
+
+[thirdparty_message_edit_disclaimer]
+other="""Conseils sur le contenu tiers"""
+
+[thirdparty_message_single_item]
+other = """&#128711; Cet élément renvoie à un projet ou un produit tiers qui ne fait pas partie de Kubernetes lui-même. <a class="alert-more-info" href="#third-party-content-disclaimer">Plus d'informations</a>"""
+
+[thirdparty_message_disclaimer]
+other = """<p>Les éléments sur cette page font référence à des produits ou projets tiers qui fournissent des fonctionnalités requises par Kubernetes. Les auteurs du projet Kubernetes ne sont pas responsables de ces produits ou projets tiers. Consultez les <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">lignes directrices du site de la CNCF</a> pour plus de détails.</p> <p>Vous devriez lire le <a href="/docs/contribute/style/content-guide/#third-party-content">guide</a> avant de proposer une modification qui ajoute un nouveau lien tiers. </p>"""
```
Comment (sftim) [inline data/i18n/fr/fr.toml:164]: (nit)
```suggestion
other = """<p>Certains éléments sur cette page font référence à des produits ou projets tiers qui fournissent des fonctionnalités requises par Kubernetes. Les auteurs du projet Kubernetes ne sont pas responsables de ces produits ou projets tiers. Consultez les <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">lignes directrices du site de la CNCF</a> pour plus de détails.</p> <p>Vous devriez lire le <a href="/docs/contribute/style/content-guide/#third-party-content">guide</a> avant de proposer une modification qui ajoute un nouveau lien tiers. </p>"""
```

I'm a bit rusty at French, but I think this captures the intent of the original wording better.

File: content/fr/docs/tasks/administer-cluster/declare-network-policy.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+* [Antrea](/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy/)
+* [Calico](/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy/)
+* [Cilium](/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy/)
+* [Kube-router](/docs/tasks/administer-cluster/network-policy-provider/kube-router-network-policy/)
+* [Romana](/docs/tasks/administer-cluster/network-policy-provider/romana-network-policy/)
+* [Weave Net](/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy/)
+
+<!-- steps -->
+
+## Créer un déploiement `nginx` et l'exposer via un service
```
Comment (sftim) [inline content/fr/docs/tasks/administer-cluster/declare-network-policy.md:27]: (nit)
```suggestion
## Créer un Deployment `nginx` et l'exposer via un Service
```

**Review by sftim — COMMENTED** [review]
_(optional to account for this feedback / I am not a reviewer for French)_

In the headings, I'd use the original (English) API kinds. I think that helps teach what the English names are, for readers who are much less strong in English than in French.

Also, a suggestion about the third party content notice.

**Comment by k0rventen** [comment]
> _(optional to account for this feedback / I am not a reviewer for French)_
> 
> In the headings, I'd use the original (English) API kinds. I think that helps teach what the English names are, for readers who are much less strong in English than in French.
> 
> Also, a suggestion about the third party content notice.

Thx, for some reason I checked that Pod was properly named, but not for svcs and deployments. I'll make a pass on the whole doc to be more consistent.

**Review by perriea — APPROVED** [review]

**Pull request merged**